app-office/gnucash/gnucash-5.4-r1.ebuild

```diff
@@ -111,6 +117,9 @@ PATCHES=(
 	# This is only to prevent webkit2gtk-4 from being selected.
 	# https://bugs.gentoo.org/893676
 	"${FILESDIR}/${PN}-5.0-webkit2gtk-4.1.patch"
+
+	# bug #919859
+	"${FILESDIR}/${PN}-5.4-fix-python-finding.patch"
```

Comment:
where's the patch?

Reply:
good catch

a important bit didnt get staged!